packages/entities/src/lib/update.mutation.spec.ts

```diff
@@ -80,4 +82,66 @@ describe('update', () => {
     store.reduce(updateAllEntities({ open: true }, { ref: UIEntitiesRef }));
     toMatchSnapshot(expect, store, 'open true');
   });
+
+  describe('Upsert', () => {
+    const updater = (e: Todo) => ({ ...e, title: 'elf' });
+    const creator = (id: number) => createTodo(id);
+    const options = {
+      updater,
+      creator,
+    };
+
+    it(`should add the entity if it doesn't exists`, () => {
+      const store = createEntitiesStore();
+      store.reduce(addEntities([createTodo(1)]));
```

Comment:
This toMatchSnapshot is redudant, you're not testing the addEntities mutation. Check the rest of the file, please.